packages/nimble-components/CONTRIBUTING.md

```diff
@@ -375,6 +375,28 @@ const nimbleButton = Button.compose({
 });
 ```
 
+If delegating focus, you must forward the `tabindex` attribute to any focusable elements in the shadow DOM. While some browsers (e.g. Chrome) will work properly without forwarding, others (e.g. Firefox) won't. Override the `tabIndex` property and mark it as an attribute:
+
+```ts
+export class MyComponent {
+    ...
+    @attr({ attribute: 'tabindex', converter: nullableNumberConverter })
+    public override tabIndex!: number;
+}
+```
+
+Then in the template, bind the focusable elements' `tabindex` to the host component's property:
+
+```html
+html<MyComponent
+    >`
+    <nimble-button ... tabindex="${x => x.tabIndex}"></nimble-button>
+    // or for an element that isn't focusable by default:
+    <div ... tabindex="${x => (x.disabled ? null : x.tabIndex ?? 0)}"></div>
```

Comment:
should tweak this snippet for the copy-pasters to incorporate feedback issues: prefer undefined, unambiguous operators, consistent typing, nullish in bindings.
Maybe it's not well known but arrow functions can be multiline (though usually a sign of doing too much work in template but potentially useful for examples)

```
<div tabindex = "${x => {
    const tabindex = x.tabIndex ?? 0;
    return x.disabled ? undefined : `${tabindex}`;
}">
```

Reply:
Also fixing mangling by prettier that I hadn't noticed.